server/src/main/java/org/elasticsearch/ingest/IngestService.java

```diff
@@ -194,7 +195,8 @@ private static Map<String, Processor.Factory> processorFactories(List<IngestPlug
                 }
             }
         }
-        return Collections.unmodifiableMap(processorFactories);
+        logger.debug("registered processor types: {}", processorFactories.keySet());
```

Comment:
super minor nit suggestion:
```suggestion
        logger.debug("registered ingest processor types: {}", processorFactories.keySet());
```

or 
```suggestion
        logger.debug("registered pipeline processor types: {}", processorFactories.keySet());
```

Reply:
Sure thing, I went with `registered ingest processor types` via 4487fc984e82b819b322dc958558669ccd125cce.